MoEngagePluginCards/Helpers/Extensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  Extensions.swift
+//  MoEngagePluginCards
+//
+//  Created by Soumya Mahunt on 20/06/23.
+//
+
+import Foundation
+
+extension UIColor {
+    func convertRGBToHex() -> String {
+        var r: CGFloat = 0
```

Comment:
@msoumya-engg-sdk I hope this should there in native

Reply:
@msoumya-engg-sdk if really required, either write moengage name or convert to function

Reply:
@RakshithaAcharya made changes